core/core_bind.cpp

```diff
@@ -919,6 +919,17 @@ Geometry3D *Geometry3D::get_singleton() {
 	return singleton;
 }
 
+Vector<Vector3> Geometry3D::compute_convex_mesh_points(const TypedArray<Plane> &planes) {
```

Comment:
Little nitpick, naming convention wise, if we have a public function `compute_convex_mesh_points` that can't be called from GDScript, we call the wrapper function that does any conversions `_compute_convex_mesh_points`.

Reply:
(and this would be a protected method so as to not confuse what to call from C++)

Reply:
I was going to rename this, but nothing else in core_bind.cpp follows that naming convention, so that would make this the outlier in this file.